src/data/nat/digits.lean

```diff
@@ -148,6 +149,30 @@ begin
   { dsimp [of_digits], rw ih, },
 end
 
+-- should I bury it in the main proof with a `have` ?
+lemma of_digits_eq_sum_map_with_index_aux (b : ℕ) (l : list ℕ) :
```

Comment:
Is this going be be useful elsewhere? If so, it might make sense to move this to another file (say one where `zip_with`, `range` and `sum` will be defined). If not, then it might make sense to mark this as `private`?

Reply:
Agreed, I'm not sure it's going to be useful elsewhere